custom-fields/custom-field-readonly.component.ts

```diff
@@ -2,14 +2,16 @@ import { Component } from "@angular/core";
 import { FieldBase } from "progress-sitefinity-adminapp-sdk/app/api/v1";
 
 @Component({
-    template: require("./custom-field-readonly.component.html"),
-    styles: [
-        `
-            .custom-input {
-                height: 100px;
-                border: 2px solid red;
-            }
-        `
-    ]
+    templateUrl: "./custom-field-readonly.component.html",
+    styles: [`
+        .custom-title-input {
```

Comment:
Maybe this should be in .css file also, just like custom-field-write.component.css ?

Reply:
I left it this way to showcase both possibilities, but good point to add comment on both places

Reply:
done